python/tvm/autotvm/task/nnvm_integration.py

```diff
@@ -54,6 +54,7 @@ class TaskExtractEnv:
     def __init__(self):
```

Comment:
We should do better decoupling here, as we will drop nnvm in the future

Move `TaskExtractEnv` to topi_integration.py,
Move all `import nnvm` to nnvm_integration.py,
Move all `import relay` to relay_integration.py.

Reply:
ok, I will try to refactor for that

Reply:
Because one operators `conv` can correspond to many topi functions.